pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -170,6 +171,7 @@ type OperatorConfigurationData struct {
 	LoggingRESTAPI             LoggingRESTAPIConfiguration        `json:"logging_rest_api"`
 	Scalyr                     ScalyrConfiguration                `json:"scalyr"`
 	LogicalBackup              OperatorLogicalBackupConfiguration `json:"logical_backup"`
+	PodAnnotations             map[string]string                  `json:"podAnnotations"`
```

Comment:
Why is there a field for `PodAnnotations` when you've already have `CustomPodAnnotations`?